controllers/conditions.go

```diff
@@ -56,15 +56,15 @@ func (r *StorageSystemReconciler) setConditionResourcePresent(instance *odfv1alp
 	}
 
 	if err == nil {
-		logger.Info("set condition", odfv1alpha1.ConditionResourcePresent, corev1.ConditionTrue)
+		logger.Info("set condition", string(odfv1alpha1.ConditionResourcePresent), string(corev1.ConditionTrue))
```

Comment:
This seems unrelated to this PR. Is this fixing an issue that was introduced before?

Reply:
yes